tests/test_project.py

```diff
@@ -122,6 +123,72 @@ def test_status_performance(self):
         assert time < 10
 
 
+class TestProjectStatusFilterOperations(TestProjectBase):
+    project_class = define_status_test_project._TestProject
+
+    def mock_project(self):
+        project = self.project_class.get_project(root=self._tmp_dir.name)
+
+        for a in range(2, 4):
+            for b in range(2):
+                job = project.open_job(dict(a=a, b=b)).init()
+                job.doc.a = a
+                job = project.open_job(dict(a=dict(a=a), b=b)).init()
+                job.doc.b = b
+
+        return project
+
+    @pytest.fixture(scope="function")
+    def project(self):
+        return self.mock_project()
+
+    @pytest.fixture(scope="function")
+    def get_status(self, project):
+        def _get_status(kwargs):
+            with redirect_stdout(StringIO()) as stdout:
+                with redirect_stderr(StringIO()) as stderr:
+                    project.print_status(**kwargs)
+                    return stdout.getvalue().split("\n"), stderr.getvalue().split("\n")
+
+        return _get_status
+
+    @pytest.mark.parametrize(
+        "groups",
+        [
+            ["group1"],
+            ["group2"],
+            ["group1", "group2"],
+            ["op1"],
+            ["group1", "op3"],
+            ["group2", "op1"],
+            ["op1", "op3"],
+        ],
+        ids=[
+            "group1",
+            "group2",
+            "group1_2",
+            "op1",
+            "group1_op3",
+            "group2_op1",
+            "op1_op3",
+        ],
+    )
+    def test_groups(self, groups, get_status):
+        stdout, _ = get_status({"operation": groups})
+        excluded_groups = {"group1", "group2", "op1", "op2", "op3"} - set(groups)
+        if len(excluded_groups) > 0:
+            # This needs to be modified if we test most than two groups
+            for excluded_group in list(excluded_groups):
+                assert excluded_group not in "".join(stdout)
```

Comment:
Define `operations_output` above, then use it here:
```suggestion
                assert excluded_group not in operations_output
```